core/pom.xml

```diff
@@ -224,6 +224,10 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
     </dependency>
+    <dependency>
```

Comment:
We used to get this transitively from Jackson; need to make it explicit now.

Reply:
ah, this one is nasty, we've had soo many conflict on this one..

Reply:
This much isn't a change to the final dependency graph, at least. It always need/depended on scala-reflect